critwall/build.gradle

```diff
@@ -21,12 +21,13 @@ dependencies {
     compile 'org.opencadc:cadc-storage-adapter:[0.3,1.0)'
     compile 'org.opencadc:cadc-util:[1.3.4,2.0)'
     compile 'org.opencadc:cadc-inventory:[0.7,2.0)'
-    compile 'org.opencadc:cadc-inventory-db:[0.6,1.0)'
+    compile 'org.opencadc:cadc-inventory-db:[0.7.4,1.0)'
     compile 'org.opencadc:cadc-registry:[1.5,2.0)'
-    compile 'org.opencadc:cadc-storage-adapter-ad:[0.1,)'
+    compile 'org.opencadc:cadc-storage-adapter-fs:[0.6,)'
+    compile 'org.opencadc:cadc-vos:[1.1.8,)'
+    compile 'org.opencadc:cadc-vosi:[1.0,)'
```

Comment:
are you sure cadc-vosi is needed here? It's generally a (poorly named) server-side lib

Reply:
(as discussed in slack) gone.